src/mem.c

```diff
@@ -129,7 +129,7 @@ coap_malloc_type(coap_memory_tag_t type, size_t size) {
 
   if (size > container->size) {
     coap_log(LOG_WARNING,
-             "coap_malloc_type: Requested memory exceeds maximum object size (type %d, size %d, max %d)\n",
+ "coap_malloc_type: Requested memory exceeds maximum object size (type %d, size %d, max %d)\n",
```

Comment:
I would like to keep the indentation here. If the line gets too long the string can be split.